apps/user-office-backend/src/factory/pdf/proposal.ts

```diff
@@ -254,3 +357,185 @@ export const collectProposalPDFData = async (
 
   return out;
 };
+
+export const collectProposalPDFDataTokenAccess = async (
```

Comment:
Forgive my lack of familarity with the new proposals system, but why do we need a separate `collectProposalPDFDataTokenAccess`? It looks like it has similar arguments, structure and response to the above `collectProposalPDFData`, and gets called from the same location, so I'm curious how they differ and why we need so much almost duplicate code

Reply:
They are different collectProposalPDFDataTokenAccess don't go through graphql  queries but get data directly from data sources.